cadl-extension/src/code-model-builder.ts

```diff
@@ -719,54 +719,44 @@ export class CodeModelBuilder {
     sealed: boolean,
   ): ChoiceSchema | SealedChoiceSchema | ConstantSchema {
     const namespace = getNamespace(type);
-    const isConstant = false; //sealed && type.members.length === 1;
```

Comment:
removed (as well as the branch code) that we don't use Cadl Enum for Java Constant